esp/esp/program/modules/handlers/commmodule.py

```diff
@@ -109,6 +110,11 @@ def commprev(self, request, tl, one, two, module, extra, prog):
         esp_firstuser = ESPUser(firstuser)
         contextdict = {'user'   : ActionHandler(esp_firstuser, esp_firstuser),
                        'program': ActionHandler(self.program, esp_firstuser) }
+
+        htmlbody = unicode(loader.get_template('email/default_email_html.txt').render(DjangoContext({'msgbdy': body,
```

Comment:
Also, I think you want `htmlbody` instead of `body` here, otherwise lines 105-108 are meaningless.  Also also, passing the user and program in here is unnecessary.